javacutil/src/main/java/org/checkerframework/javacutil/PluginUtil.java

```diff
@@ -34,11 +34,31 @@
     public static final String JAVAC_PATH_OPT = "-javacJar";
 
     /**
-     * Option name for specifying an alternative jdk.jar location. The accompanying value MUST be
-     * the path to the jar file (NOT the path to its encompassing directory)
+     * Option name for specifying an alternative location for the annotated JDK.
+     *
+     * <p>In case of Java 8, an alternative jdk.jar location. The accompanying value MUST be the
+     * path to the jar file (NOT the path to its encompassing directory).
+     *
+     * <p>In case of Java 9+, a directory containing annotated &lt;module&gt;.jar. The supplied
```

Comment:
How about:
```
     * <p>In case of Java 9+, a directory containing annotated JDK modules. The supplied
     * directory MUST contain a {@code Patch_Modules_argfile} file containing {@code --patch-module
     * <jdkModule>=<annotatedModule>} directives, one per line. The string `${CURRENT_JDK_FOLDER}` will
     * be replaced with this directory. The directory must also contain the refrenced &lt;module&gt;.jar files.
```

However, this seems like an odd place to document the file format. So move this somewhere else and refer to it here.

Reply:
Done.